Assets/Scripts/Robot/IRobotController.cs

```diff
@@ -77,6 +77,8 @@ public interface IRobotController
         /// </summary>
         float GlobalAngle { get; }
 
+        RobotConstraints Constraints { get; }
```

Comment:
Expose the property you need from it instead, because RobotConstriants is forbidden knowledge.